packages/ui-kit/src/Toggler/index.tsx

```diff
@@ -5,7 +5,12 @@ import type { ChangeEvent } from 'react';
 interface Toggler {
   id: string;
   name?: string;
-  label?: React.ReactChild | React.ReactChild[];
+  label?: React.ReactNode;
+  leftLabel?: React.ReactNode;
```

Comment:
`leftLabel` does not reflect that this label is used for `false` value. 
Can we maybe rename it to something like `negativeLabel`, `offLabel` etc?